api/etcdserverpb/gw/rpc.pb.gw.go

```diff
@@ -47,7 +47,7 @@ func request_KV_Range_0(ctx context.Context, marshaler runtime.Marshaler, client
 	}
 
 	msg, err := client.Range(ctx, &protoReq, grpc.Header(&metadata.HeaderMD), grpc.Trailer(&metadata.TrailerMD))
-	return msg, metadata, err
+	return protov1.MessageV2(msg), metadata, err
```

Comment:
This hack looks good. I was think about generating two proto codes. For the gateway one, we can use new API code. And for the main grpc protocal, we can use gogo.